File: data/src/main/java/com/platdmit/data/api/models/ApiAverageCPU.java
package com.platdmit.data.api.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ApiAverageCPU {
    @SerializedName("total")
    @Expose
    public String total;
    @SerializedName("now")
    @Expose
    public String now;
    @SerializedName("hourly")
    @Expose
    public double hourly;

    public String getTotal() {
        return total;
    }

    public void setTotal(String total) {
        this.total = total;
    }

    public String getNow() {
        return now;
    }

    public void setNow(String now) {
        this.now = now;
    }

    public double getHourly() {
        return hourly;
    }

    public void setHourly(double hourly) {
        this.hourly = hourly;
    }
}

File: feature_servers/src/main/java/com/platdmit/feature_servers/data/retrofit/models/ApiAverageCPU.kt
package com.platdmit.feature_servers.data.retrofit.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ApiAverageCPU(
        @SerializedName("total")
        @Expose
        val total: String,

        @SerializedName("now")
        @Expose
        val now: String,

        @SerializedName("hourly")
        @Expose
        val hourly: Double
)